### PR TITLE
added best practice for crate setup on ec2

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: docs/best_practice/ec2_setup.txt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+Basic Configuration
+===================
+
+First, to activate EC2 discovery **simply set the discovery type to ``ec2``**::
+
+  discovery.type: ec2
+
+The following settings allow you to control EC2 discovery once it is enables.
```
Comment (bogensberger) [inline docs/best_practice/ec2_setup.txt]: enabled

File: docs/best_practice/ec2_setup.txt [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Authentication
+--------------
+
+In order to be able to use the EC2 API, the Crate must `sign the requests`_ using
```
Comment (bogensberger) [inline docs/best_practice/ec2_setup.txt]: s/the Crate/Crate

**Comment by bogensberger** [comment]
i'm not sure if the conclusion is really useful, but it's ok to let it there.
otherwise ack

**Comment by chaudum** [comment]
@bogensberger agree, I removed the "conclusion"

**Pull request merged**